docs/_policies/services.md

```diff
@@ -4,57 +4,38 @@
 
 The [DARC team](https://gsbresearchhub.stanford.edu/support-units/darc) provides a range of services to support research at the Graduate School of Business, and has developed specific request forms for common issues.
 
-## Service Overview
+### Service Overview
```

Comment:
I think Other infrastructure and Links to other services can be moved after the darc services or after Feedback? I also saw a few repetitive sentences introducing DARC. We can probably just do one sentence linking to DARC at the beginning and cut the rest.

Reply:
done